lib/octicons_react/src/createIconComponent.js

```diff
@@ -12,24 +12,31 @@ export function createIconComponent(name, defaultClassName, getSVGData) {
 
   function Icon({
     'aria-label': ariaLabel,
+    'aria-labelledby': arialabelledby,
     tabIndex,
     className = defaultClassName,
     fill = 'currentColor',
     size = 16,
+    ref,
+    id,
+    title,
     verticalAlign = 'text-bottom'
   }) {
     const height = sizeMap[size] || size
     const naturalHeight = closestNaturalHeight(heights, height)
     const naturalWidth = svgDataByHeight[naturalHeight].width
     const width = height * (naturalWidth / naturalHeight)
     const path = svgDataByHeight[naturalHeight].path
+    const svgProps = {...(id ? {id} : {}), ...(ref ? {ref} : {})}
```

Comment:
With the changes to `ref` I bet we could remove `svgProps` here and just set `id` explicitly on `<svg>` below